test/testutil/helper.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020-2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "../testutil.h"
+
+time_t test_asn1_string_to_time_t(const char *asn1_string)
+{
+    ASN1_TIME *timestamp_asn1 = NULL;
+    struct tm *timestamp_tm = NULL;
+#if defined(__DJGPP__)
+    char *tz = NULL;
+#elif !defined(USE_TIMEGM)
```

Comment:
This `USE_TIMEGM` thing is defined in the original `a_time.c`, so you need either move that definition lines here, or just `defined(__FreeBSD__) || defined(__wasi__)`

Reply:
Also the timezone definition for Windows must be moved.